### PR TITLE
Change single quote to double quote

### PR DESCRIPTION
**Pull request by gravidesaldrin**
Hi Sir. Please review this PR. Thank You.

### CODE REVIEW

File: Gemfile [hunk 1]
```diff
@@ -1,47 +1,38 @@
...
+gem "sass-rails", "5.0.2"
+gem "uglifier", "2.5.3"
+gem "coffee-rails", "4.1.0"
+gem "jquery-rails", "4.0.3"
+gem "turbolinks", "2.3.0"
+gem "jbuilder", "2.2.3"
+gem "sprockets-rails", "3.0.0"
+gem "sdoc", "0.4.0", group: :doc
+gem "faker", "1.4.2"
+gem "will_paginate", "3.0.7"
```
Comment (dieunb) [inline Gemfile]: `gem "kaminari"` instead of that

File: Gemfile [hunk 2]
```diff
@@ -1,47 +1,38 @@
...
+gem "bootstrap-will_paginate", "0.0.10"
```
Comment (dieunb) [inline Gemfile]: same

File: Gemfile [hunk 3]
```diff
@@ -1,47 +1,38 @@
...
+gem "carrierwave", "0.10.0"
+gem "mini_magick", "3.8.0"
+gem "fog", "1.36.0"
 
 group :development, :test do
-  # Call 'byebug' anywhere in the code to stop execution and get a debugger console
-  gem 'byebug'
+  gem "sqlite3", "1.3.9"
+  gem "byebug", "3.4.0"
```
Comment (dieunb) [inline Gemfile]: `gem "pry"`

**Comment by gravidesaldrin** [comment]
Ok Sir. Noted

**Pull request merged**